src/Classes/Cart.php

```diff
@@ -45,5 +47,39 @@ function checkId($id)
         }
     }
 
+    function emailItems()
```

Comment:
I'm being nitpicky here, but every function should have a visibility declared; by default, PHP will imply `public` if one isn't offered, but it's best practice to always include `public`, `protected`, or `private` respectively on class functions.

Let me know if you don't know what this means